packages/css/src/dialog/dialog.scss

```diff
@@ -0,0 +1,107 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../utils/breakpoint";
+
+.amsterdam-dialog {
+  align-content: start;
+  background-color: var(--amsterdam-dialog-background-color);
+  border: var(--amsterdam-dialog-border);
+  display: grid;
```

Comment:
Overriding the default display property here makes a closed `dialog` accessible for screenreader, which we don't want. `pointer-events: none` isn't enough to fix that.